lib/ask_web/views/channel_view.ex

```diff
@@ -20,7 +20,8 @@ defmodule AskWeb.ChannelView do
       channelBaseUrl: channel.base_url,
       settings: channel.settings,
       patterns: channel.patterns,
-      status_info: channel.status
+      status_info: channel.status,
+      userEmail: channel.user_email
```

Comment:
Regarding the casing, we normally follow [this snake_case convention](https://hexdocs.pm/elixir/main/naming-conventions.html#casing) in the backend in elixir files and then the standard javascript camelCase convention in the front-end. We use the `camelize` function of the `humps` library to convert the keys [here](https://github.com/instedd/surveda/blob/01cd890c1f6031a0644044523a2311b43eea84b1/assets/js/api.js#L617).

Reply:
Elixir follows Ruby: `CamelCase` are constants (e.g. modules), `lower_cased` are functions and variables, and `snakeCase` should not exist on the backend (only in JavaScript).